v3/tutorials/02-proof-of-existence/index.mdx

```diff
@@ -540,11 +526,10 @@ This React component enables you to expose the proof-of-existence capabilities a
              />
              {/* Button to revoke a claim. Only active if a file is selected, and is already claimed. Updates the `status`. */}
              <TxButton
-               accountPair={accountPair}
                label="Revoke Claim"
                setStatus={setStatus}
                type="SIGNED-TX"
-               disabled={!isClaimed() || owner !== accountPair.address}
```

Comment:
This was failing for me, sadly. Tested here: https://github.com/substrate-developer-hub/substrate-front-end-template/blob/ds/poe-example/src/TemplateModule.js